subprojects/core/src/main/java/org/gradle/api/internal/changedetection/state/ZipHasher.java

```diff
@@ -72,6 +98,7 @@ public void appendConfigurationToHasher(Hasher hasher) {
         hasher.putString(getClass().getName());
         resourceHasher.appendConfigurationToHasher(hasher);
         resourceFilter.appendConfigurationToHasher(hasher);
+        attributeResourceFilter.appendConfigurationToHasher(hasher);
```

Comment:
We are missing the property filter here.

Reply:
Added.